app/assets/scripts/components/status-api.js

```diff
@@ -1,9 +1,32 @@
 'use strict';
 import React from 'react';
 
+import config from '../config';
+
 var StatusApi = React.createClass({
   displayName: 'StatusApi',
 
+  getInitialState: function () {
+    return {
+      health: null
+    }
+  },
+
+  componentDidMount: function () {
+    fetch(`${config.api}status`)
```

Comment:
I prefer to have no trailing slash in the config so you can do `${config.api}/status`. But it's more a cosmetic thing.